chancloser.go

```diff
@@ -427,6 +427,14 @@ func (c *channelCloser) ProcessCloseMsg(msg lnwire.Message) ([]lnwire.Message, b
 		}
 		c.closingTx = closeTx
 
+		// Before closing, we'll attempt to send a disable update for
+		// the channel. We do so before closing the channel as otherwise
+		// the current edge policy won't be retrievable from the graph.
+		if err := c.cfg.disableChannel(c.chanPoint); err != nil {
```

Comment:
Is there a reason for this not being able to stay in the same place (after `MarkCommitmentBroadcasted`)?

Reply:
the disable should occur before broadcasting the transaction, as there is an unlikely case where the txn gets instamined and the edge is no longer in our database to disable.

Looking at this though, I would say we should mark commitment broadcast before broadcasting the transaction, and then reliably publish on restarts. I would also venture to say this should be marked with a different status since we aren't actually broadcasting a commitment, cc @Roasbeef

Reply:
I think this case is distinct from the force close case, as a force close means we don't want to continue with the channel at all (either HTLC timeout or we want to exit nao), vs a cooperative channel closure, which if really needed, should be re-attempted once both nodes are active again.